src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1960,9 +1967,9 @@ def list_locations(cmd, sku, linux_workers_enabled=None):
 def _check_zip_deployment_status(deployment_status_url, authorization, timeout=None):
     import requests
     import time
-    total_trials = (int(timeout) // 30) if timeout else 10
+    total_trials = (int(timeout) // 30) if timeout else 300
     for _num_trials in range(total_trials):
-        time.sleep(30)
```

Comment:
This is to handle the perceived perfomance - we increaded the overall default timeout &decrease the polling interval. In cases where the deployment was taking 17s, we are waiting & additional ~13s before zip deployment exists & shows a success.